### DISCUSSION TITLE
Create static resource command

### DISCUSSION THREAD

**Discussion by atnmorrison** [forcedotcom/salesforcedx-vscode#5472] [Ideas]
I would like to be able to create static resources within vs code. There are commands for most components but this one is missing. In lightning if we're developing a js library for our components it would make things much easier.

Even better would be if it could  support zip files and automatically zip and push the static resources zipped while being able to easily edit in the editor

**Comment by atnmorrison** [comment]
seems like the ability to handle zipped static resources is already there, only thing missing is being able to create a new static resource without having to manually create it in the scratch org and pull it down

**Comment by dschach** [comment]
I believe this has been delivered. Am I wrong? @smaddox-sf

**Comment by mohanraj-r** [comment]
@AnanyaJha @atnmorrison Is this issue still relevant? If not can we close it?

**Comment by RupertBarrow** [comment]
We still need this, in the following form : 
- right click on a ".../staticresources" folder : display menu "SFDX: create new static resource", ask for the name of the resource to be create, and call 'sfdx force:staticresource:create -MyStaticResource'
- command palette : add the "SFDX: create new static resource" command

cc @dschach @smaddox-sf @mohanraj-r

**Comment by AnanyaJha** [comment]
Hi @atnmorrison @RupertBarrow - checking in to see if this is still functionality y'all are looking for? Thanks so much, this'll help with prioritizing our future work!

**Reply by RupertBarrow** [reply]
Yes please !

**Comment by dschach** [comment]
Yes. Being able to create, essentially, a folder in the CLI that I can put javascript or image files into is definitely something that would be easier to do via CLI than via making something in the scratch org and retrieving it.
